src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/events/OutboundEventsPublisher.kt

```diff
@@ -0,0 +1,51 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services.court.events
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import software.amazon.awssdk.services.sns.model.MessageAttributeValue
+import software.amazon.awssdk.services.sns.model.PublishRequest
+import uk.gov.justice.digital.hmpps.whereabouts.config.Feature
+import uk.gov.justice.digital.hmpps.whereabouts.config.FeatureSwitches
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+
+@Service
+class OutboundEventsPublisher(
```

Comment:
Some comments/deprecations on this classes?  They will be removed after migration is done?

Reply:
Its using SQS lib v4.1.0 - latest is 4.3.5 - wondering if there's value in updating the version - and if so, in a different, dedicated PR?